src/main/java/frc/robot/commands/swervedrive/drivebase/TeleopDrive.java

```diff
@@ -68,9 +69,24 @@ public void initialize()
   @Override
   public void execute()
   {
-    double xVelocity   = Math.pow(vX.getAsDouble(), 3);
-    double yVelocity   = Math.pow(vY.getAsDouble(), 3);
-    double angVelocity = Math.pow(omega.getAsDouble(), 3);
+     double angVelocity;
+     double xVelocity;
+     double yVelocity;
+    if(DriverStation.getAlliance().isPresent() && DriverStation.getAlliance().get() == DriverStation.Alliance.Blue)
```

Comment:
Use a value that is -1 or +1 based on the logic, and multiply the x and y velocity.  Then you don't need to duplicate the logic.